logstash.conf.m4

```diff
@@ -29,16 +31,16 @@ output {
 }
 filter {
  ruby {
-   code => "event['timestamp'] /= 1_000_000"
```

Comment:
I prefer replacing the original field ... do you really need the nanoseconds or microseconds?